staging/src/k8s.io/api/node/v1/types.go

```diff
@@ -50,6 +50,8 @@ type RuntimeClass struct {
 	// in a pod.
 	// The Handler must be lowercase, conform to the DNS Label (RFC 1123) requirements,
 	// and is immutable.
+	// +k8s:format="k8s-short-name"
+	// +k8s:immutable
```

Comment:
There's a push to make all fields either required or optional - I assume this should be required?